### PR TITLE
Mirror mk2 + symmetrize

### PR DESCRIPTION
**Pull request by portnov**
This is a third attempt on "mirror mk2" node, plus "symmetrize" node. Previous efforts: #2466 , #2586 .

symmetrize_3

## Preflight checklist

Put an x letter in each brackets when you're done this item:

- [x] Code changes complete.
- [x] Code documentation complete.
- [x] Documentation for users complete (or not required, if user never sees these changes).
- [x] Manual testing done. 
- [ ] Unit-tests implemented.
- [x] Ready for merge.

### CODE REVIEW

**Comment by portnov** [comment]
Screenshot_20191215_165556

**Comment by portnov** [comment]
Screenshot_20191215_171124

File: nodes/modifier_change/symmetrize.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            name = "Merge Distance",
+            description = "maximum distance for merging. does no merging if 0",
+            default = 0.001, min = 0,
+            update = updateNode)
+
+    def draw_buttons(self, context, layout):
+        layout.prop(self, "direction")
+        layout.prop(self, "merge_dist")
+
+    def get_symmetrize_direction(self):
```
Comment (zeffii) [inline nodes/modifier_change/symmetrize.py]: def get_symmetrize_direction(self):
    return self.direction[1]

**Comment by portnov** [comment]
Screenshot_20191215_231239

**Comment by portnov** [comment]
Screenshot_20191215_232123

**Comment by portnov** [comment]
Screenshot_20191215_234157

File: nodes/modifier_change/symmetrize.py (hunk 1 above)
Comment (portnov) [reply nodes/modifier_change/symmetrize.py]: This is shorter, but I find it less explicit.

**Comment by Durman** [comment]
Is it possible to symmetrize mesh without faces?

**Comment by portnov** [comment]
idk, try it :)

**Pull request merged**